ui/src/components/TFOption.vue

```diff
@@ -896,6 +989,25 @@ export default {
         ...this.data,
       }
 
+      // convert to json to AI Report
```

Comment:
I See this mapping logic while saving the file. Where is the logic while reading the file? Could you make a new function for that mapping that is able to handle reading/writing the keys properly?

Reply:
added and tested the read/write functionality of AI reports in TFSetting.